app/lib/actions/katello/organization/manifest_import.rb

```diff
@@ -23,7 +23,7 @@ def plan(organization, path, force)
                 plan_action(Katello::Repository::RefreshRepository, repo)
               end
             end
-            plan_self
+            plan_self(candlepin_task: cp_import_task.output[:task])
```

Comment:
i don't think this 'candlepin_task' passing around is needed anymore, right?  (i see it being passed through to other actions, but its not really used)

Reply:
I just really like seeing the candlepin task details on the Tasks page.

Reply:
I suppose it's not needed in both input and output though

Reply:
Problem is there's no way to have it just in output, which would make the most sense.  In order to get to output it has to be passed.

Reply:
you dig into the dynflow console to get at the subtask details and see it there, is that sufficient?

Reply:
For a developer I guess that's fine, but users would rarely if ever go into the dynflow console.  I feel like it's a better user experience to leave it in.

Reply:
normal users probably aren't going to go looking for a candlepin task either i wouldn't think.  I ask this because we generate these task in sub-actions in dozens of places and we don't generally propagate task data up, so this would be an exception to what we do elsewhere.

Reply:
Okaaaaay, removing.  :cry:

Reply:
Added https://projects.theforeman.org/issues/31876 as an RFE

Reply:
@jlsherrill, see @adamruzicka comment here:  https://projects.theforeman.org/issues/31876#change-145787

Do you think it's possible for me to use that presenter in a way that would both give the better user experience and be consistent with katello code?

Reply:
what kind of information from the task is it you're interested in exposing?

Reply:
looking at the task output:

```
---
task:
  created: '2021-02-15T20:47:37+0000'
  updated: '2021-02-15T20:48:11+0000'
  id: 4028fa387792895c0177a772d8b300ec
  name: Import Manifest
  group: 
  origin: devel.balmora.example.com
  executor: devel.balmora.example.com
  principal: foreman_admin
  state: FINISHED
  previousState: RUNNING
  startTime: '2021-02-15T20:47:37+0000'
  endTime: '2021-02-15T20:48:11+0000'
  attempts: 1
  maxAttempts: 1
  statusPath: "/jobs/4028fa387792895c0177a772d8b300ec"
  resultData:
    created: 1613422091232
    updated: 1613422091232
    id: 4028fa387792895c0177a7735be03445
    status: SUCCESS_WITH_WARNING
    statusMessage: Default_Organization file imported successfully.One or more inactive
      subscriptions found in the file.
    fileName: import_b51c6c50c540001a6402.zip
    generatedBy: rhn-engineering-jsherrill
    generatedDate: 1598646848281
    upstreamConsumer:
      created: 1613422091232
      updated: 1613422091232
      id: 4028fa387792895c0177a7735be03446
      uuid: dc13b59f-2d6c-4072-930e-631abfd3c08b
      name: jsherrill
      type:
        created: 1613422057938
        updated: 1613422057938
        id: 4028fa387792895c0177a772d9d200ed
        label: satellite
        manifest: true
      ownerId: 4028fa387792895c0177928eb7630001
      contentAccessMode: 
      webUrl: access.redhat.com/management/subscription_allocations/
      apiUrl: https://subscription.rhsm.redhat.com/subscription/consumers/
  key: ImportJob
```

maybe the status message?

Reply:
here's an example of looking up a sub action and passing that as a presenter, in this case it uses the presenter of a sub-action: 
https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/sync.rb#L102-L107

you could also just grab the output directly iirc, if you want, for example:

all_planned_actions(Pulp::Repository::Sync).first.output

Reply:
It turns out the two pieces of data I would want to include in the output are the same ones currently shown in Manifest History (Status and Message).  So after today's journey of discovery I think I'm going to just leave the UI as-is. :man_shrugging: